File: week1/pF.java
package week1;

import java.util.*;

public class pF {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int min = sc.nextInt();
        int hours = 00;
        int mins = 00;
        mins = min % 60;
        hours = 12 + min / 60;
        if (hours > 24)
            hours = hours % 24;
        System.out.printf(hours + ":" + "%02d%n", mins);
        sc.close();
    }
}

File: week1/pF.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    long long min, mins, hours;
    cin >> min;
    min %= 1440;
    mins = min % 60;
    hours = (12 + min / 60) % 24;

    cout << hours/10 << hours%10 << ":" << mins/10 << mins%10 << endl;
}
